integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/QuarkusCodestartGenerationTest.java

```diff
@@ -519,13 +519,11 @@ private void checkNoExample(Path projectDir) {
     private void checkDockerfiles(Path projectDir) {
         assertThat(projectDir.resolve(".dockerignore")).exists();
         assertThat(projectDir.resolve("src/main/docker/Dockerfile.jvm")).exists()
-                .satisfies(checkContains("mvn package"))
```

Comment:
Why this check was removed?

Reply:
Good catch!

Reply:
ah because it depends on the buildtool now :)

Reply:
In this case, the `checkDockerfiles` should have an extra parameter for the expected build tool command. I think it's nice to still have that check

Reply:
+1 yeah I guess it should be fairly easy

Reply:
Yes you are right, I will add checks depending on the buildtool